### ISSUE TITLE
[Bug]: Server apotheosis axe config issue

### ISSUE THREAD

**Issue by Xtrullor** [AllTheMods/ATM-9#1514]
### Possible Fixes

Yes

### Modpack Version

2.53

### What happened?

Axes when playing on a server are being treated as light weapons when they should be treated as heavy weapons from the apotheosis mod

**Comment by TheLastPaladin** [comment]
Can confirm.

**Comment by Baylem** [comment]
Can confirm and this issue is starting to affect our multiplayer playthrough

**Comment by Xynrati** [comment]
I heard that relogging fixes this, and it's a visual bug? Alternatively, I'm having a similar issue with just about any reforge on the latest versions of ATM9/ATM9NF on my dedicated server. In both cases versions, when trying to do the axes, as noted above, it doesn't result in the desired outcome. However, I've also noticed it on pickaxes and possibly swords, specifically with the larger mining (5x3 for example) or loot pinata not be applied properly. Other parts of hte affixes, such as being able to place torches, comes across fine, but never the expanding mining area.

I'm unsure if it's the same issue as this bug, and I'm awaiting Shadow's response before making a dedicated bug for the non-Axe/Heavy specific issue I've noticed.
